opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -592,13 +601,14 @@ def __init__(
                 "larger than the recommended value of 20",
                 self._max_scale,
             )
-        self._mapping = LogarithmMapping(self._max_scale)
+        self._mapping = _new_exponential_mapping(self._max_scale)
```

Comment:
in case if `self._max_scale` is negative we may need to use a different mapping

Reply:
Right :+1: